spec/support/feature_spec_helpers.rb

```diff
@@ -65,6 +83,22 @@ def expect_page(klass)
   def snapshot
     screenshot_and_open_image
   end
+
+  private
+
+  def load_page_with_retries(page, args, attempts: 0)
```

Comment:
Attempts as name & being equal to `0` is misleading, one expects that it indicates the number of attempts to be made. Also, is anti-intuitive if someone call the method with `attempts=2` which will lead to `3-2` attempts.

Reply:
Oh my, it's definitely confusing. I changed it to be the number of attempts!